librz/bin/bin.c

```diff
@@ -1114,15 +1116,16 @@ RZ_API RZ_OWN RzList /*<RzBinMap *>*/ *rz_bin_maps_of_file_sections(RZ_NONNULL R
  * See also rz_bin_maps_of_file_sections() for the inverse, when no additional
  * sections should be added.
  * */
-RZ_API RzList /*<RzBinSection *>*/ *rz_bin_sections_of_maps(RzList /*<RzBinMap *>*/ *maps) {
+RZ_API RzList /*<RzBinSection *>*/ *rz_bin_sections_of_maps(RzPVector /*<RzBinMap *>*/ *maps) {
```

Comment:
You might as well convert the output of this function too, it has only two users.